package/jailer.sh

```diff
@@ -11,7 +11,7 @@ fi
 
 # Build the jail directory structure
 mkdir -p /opt/jail/$NAME/dev
-mkdir -p /opt/jail/$NAME/etc/ssl/certs
+mkdir -p /opt/jail/$NAME/etc/ssl
```

Comment:
> Finally, the jailer also copies the certificate information to /etc/ssl instead of /etc/ssl/certs because the Go libraries expect the certs to be there.

Confused why this change affects go libraries. This is all within the jail, the certs themselves are still being pulled out of the same place.

Reply:
The crypto package looks looks in different directories based on the OS: https://golang.org/src/crypto/x509/root_linux.go

Even though the run of `rancher-machine` is jailed, it will still look in the corresponding jailed directory for the certs.

Reply:
Ahh, so we are actually moving this because the crypto library looks in a different spot.